samcli/lib/package/artifact_exporter.py

```diff
@@ -156,6 +159,11 @@ def upload_local_artifacts(resource_id, resource_dict, property_name, parent_dir
 
     raise exceptions.InvalidLocalPathError(resource_id=resource_id, property_name=property_name, local_path=local_path)
 
+def resource_not_packageable(resource_dict):
+    inline_code = jmespath.search("InlineCode", resource_dict)
+    if inline_code is not None:
```

Comment:
can we simply do `return inline_code is not None`?

Reply:
We could at this point, wrote it this way with the idea that more such checks would follow. Also added a `make black` refactor just now.